code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -222,12 +222,18 @@
 	item_state = "coffee"
 	spillable = TRUE
 
-/obj/item/reagent_containers/food/drinks/mug/on_reagent_change(changetype)
+/obj/item/reagent_containers/food/drinks/mug/tea/on_reagent_change(changetype)
```

Comment:
```suggestion
/obj/item/reagent_containers/food/drinks/mug/on_reagent_change(changetype)
```
Otherwise only tea and coco mugs will update on reagent change.

Reply:
Updated.